deeptools/heatmapper.py

```diff
@@ -804,15 +819,28 @@ def getRegionsAndGroups(regions_file, onlyMultiplesOf=1,
 
             previnterval = ginterval
 
+            ginterval.group_idx = group_idx
             regions.append(ginterval)
             includedintervals += 1
 
         # in case we reach the end of the file
         # without encountering a hash,
         # a default name is given to regions
-        if len(regions) > group_boundaries[-1]:
+        if len(group_labels) == 0:
```

Comment:
Not in this case, we want to test that the group_labels is empty. But I just checked and the pythonic way is `if not group_labels` I will change that.